modules/config/krb5.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, pkgs, config, options, ...
+}:
+
+let
+  cfg = config.nixathena.config.krb5;
+in
+{
+  options.nixathena.config.krb5 = (let
+    mkEnableOption = lib.mkEnableOption;
+  in {
+    enable = mkEnableOption "Nixathena krb5 configuration";
+  });
+
+  config = lib.mkIf cfg.enable {
```

Comment:
This is a pretty wordy option name (and the `.config.` might cause some confusion in the NixOS module system). Maybe this could just be

```nix
{
  options.athena.krb5.enable = lib.mkEnableOption "Configure Kerberos for ATHENA.MIT.EDU";
  config = lib.mkIf config.athena.krb5.enable {
  ...
```

I don't think either of the `let` statements are doing anything here, since they're both used exactly once.